students/12th/sanghomoon/main.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+
+    <!-- open-graph -->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Westagram">
+    <meta property="og:title" content="Instagram clone page">
+    <meta property="og:description" content="Instagram clone page">
+    <meta property="og:image" content="img/instagram.png">
+    <meta property="og:url" content="#">
+
+    <!-- favicon -->
+    <link rel="icon" href="img/favicon.png">
+    <link rel="shortcut icon" href="img/favicon.png">
+
+    <title>Instagram</title>
+
+    <!-- css -->
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <nav>
+      <div class="nav-container">
+        <div class="nav-mainLogo">
+          <a href="main.html"><img src="img/logo_text.png" alt="main_logo"></a>
+        </div>
+
+        <div class="nav-searchBox">
+          <img src="img/search-icon.png" alt="search-icon">
+          <input type="search" placeholder="검색">
+        </div>
+
+        <ul class="nav-menu"> 
+          <li><img src="img/home.png" alt=""></li>
```

Comment:
```suggestion
          <li>
            <img src="img/home.png" alt="">
          </li>
```
- 태그들의 부모 자식 관계가 잘 드러나도록 줄바꿈 해주세요
- alt 솏ㅇ 빼놓지 말고 꼭 적어주세요!

Reply:
네!